src/main/java/pl/javastart/movieclub/domain/movie/MovieService.java

```diff
@@ -23,22 +26,24 @@ public class MovieService {
     private final GenreRepository genreRepository;
     private final FileStorageService fileStorageService;
 
-
-    public List<MovieDto> findAllPromotedMovies() {
-        return movieRepository.findAllByPromotedIsTrue().stream()
-                .map(MovieDtoMapper::map)
-                .toList();
+    public Page<MovieDto> findAllPagedPromotedMovies(int pageNo, int pageSize){
+        Pageable pageable = PageRequest.of(pageNo - 1, pageSize);
```

Comment:
tutaj chyba powinno byc przekazywane normalnie pageNo, bez -1

Reply:
Generalnie to założyłem sobie, że z FE defaultowa wartość dla pageNo to będzie 1. Jedynkę przekazuję z controllerów do warstw service i tam odejmuję 1, bo pageable zaczyna liczyć od 0 :) Chyba że powinienem odejmować jedynkę w innym miejscu?